packages/@storefront/core/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Added
+  - SF-1163: Add hardcoded `GBI_EVENT` to the metadata of all tracking events.
```

Comment:
Remove indentation

Reply:
Perhaps add a short explanation on what `GBI_EVENT` is for.

Reply:
Updated.